src/benchmarks/micro/runtime/System.Reflection/Invoke.cs

```diff
@@ -265,6 +327,33 @@ public void Field_Set_class()
             }
         }
 
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void Field_SetStatic_class()
+        {
+            for (int i = 0; i < Iterations; i++)
+            {
+                s_staticField_class.SetValue(s_MyClass, 42);
```

Comment:
Just curious, for static fields if setting the value with null object make any difference for perf

Reply:
The perf is the same; it is not used or validated. Thanks for noticing that - will just change to null.